### ISSUE TITLE
Update asterales end-to-end test instructions to handle subproblem decomposition and recent incompatible treemachine changes

### ISSUE THREAD

**Issue by jar398** [OpenTreeOfLife/germinator#16]
It's been a while since the asterales system worked. I still think it's a good idea and I'm hoping that other people will use it as an interface test.

I plan to raise parallel issues for treemachine and perhaps otcetera, but probably the instructions should be revised first, so that it's clear what works and what doesn't.

**Comment by josephwb** [comment]
Right. We are currently very bound to subproblem loading (rather than nexsons). We can provide the subproblems, or something more substantial will need to be done (making sure nexson related code works, perhaps generating subproblems on the fly, etc.).
